### PR TITLE
Infra agent support for arm64

### PR DESCRIPTION
**Pull request by josemore**
### Give us some context

* Infrastructure agent will be available for arm64 machines with standard linux package setup.
* This PR updates the manual installation steps and support matrix to explain arm64 support.
* Support is limited to infra-agent and few additional integrations (no log-fwd, no OHIs at the moment). 

### Are you making a change to site code?

If you're changing site code (rather than the content of a doc), please follow
[conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

### CODE REVIEW

**Comment by josemore** [comment]
We can proceed with review and merge, see https://github.com/newrelic/infrastructure-agent/releases/tag/1.18.0

**Review by x8a — APPROVED** [review]
Apart from these two comment, everything looks great. Thanks!

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx [hunk 1]
```diff
@@ -264,6 +264,9 @@ To install the infrastructure monitoring agent in Linux, follow these instructio
        ```
        printf "deb [arch=amd64] https://download.newrelic.com/infrastructure_agent/linux/apt buster main" | sudo tee -a /etc/apt/sources.list.d/newrelic-infra.list
        ```
+       <Callout variant="tip">
+         Use [arch=arm64] to install on ARM architectures such us AWS Graviton
```
Comment (x8a) [inline src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx:268]: I'd add a period at the end of the sentence. That is: `Use [arch=arm64] to install on ARM architectures such us AWS Graviton.`

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx [hunk 2]
```diff
@@ -299,87 +302,96 @@ To install the infrastructure monitoring agent in Linux, follow these instructio
        ```
        printf "deb [arch=amd64] https://download.newrelic.com/infrastructure_agent/linux/apt focal main" | sudo tee -a /etc/apt/sources.list.d/newrelic-infra.list
        ```
+       <Callout variant="tip">
+         Use [arch=arm64] to install on ARM architectures such us AWS Graviton
```
Comment (x8a) [inline src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx:306]: Same as above, `Use [arch=arm64] to install on ARM architectures such us AWS Graviton.`

**Pull request merged**